test/suite/hook_disagg.fail

```diff
@@ -0,0 +1,152 @@
+# This file is used by evergreen.yml when running the python test suite with "--hook disagg" (see hook_disagg.py).
```

Comment:
Unnecessary change.

Reply:
Good catch. Removed in [5aae60f](https://github.com/wiredtiger/wiredtiger/pull/12392/commits/5aae60f91bb6378a79307071932bc374d8444524)